fdbserver/storageserver.actor.cpp

```diff
@@ -2360,6 +2360,7 @@ ACTOR Future<Void> fetchKeys( StorageServer *data, AddingShard* shard ) {
 					if (debug_getRangeRetries >= 100) {
 						data->cx->enableLocalityLoadBalance = false;
 						// TODO: Add SevWarnAlways to say it was disabled.
```

Comment:
remove this comment since you add the event.

Reply:
Done.